source/precalculus/source/06-TR/03.ptx

```diff
@@ -12,6 +12,618 @@
+      </p>
+    </introduction>
+
+    <task>
+      <statement>
+        <p>
+          In triangle <m>ABC</m>, <m>B=37</m><degree/> and <m>a=11</m>. Which trigonometric function will best help determine the length of side <m>c</m>?
+          <ol marker= "A." cols="1">
+            <li> sine </li>
+            <li> cosine </li>
+            <li> tangent </li></ol></p>
+      </statement>
+      <answer>
+        <p>
+          B
+        </p>
+      </answer>
+    </task>
+
+    <task>
+      <statement>
+        <p>
+          In triangle <m>ABC</m>, <m>A=32</m><degree/> and <m>b=13</m>. Which trigonometric function will best help determine the length of side <m>a</m>?
+          <ol marker= "A." cols="1">
+            <li> sine </li>
+            <li> cosine </li>
+            <li> tangent </li></ol></p>
+      </statement>
+      <answer>
+        <p>
+          C
+        </p>
+      </answer>
+    </task>
+  
+    <task>
+      <statement>
+        <p>
+          In triangle <m>ABC</m>, angle <m>A=24</m><degree/> and the hypotenuse is <m>14</m>. Which trigonometric function will best help determine the length of side <m>a</m>?
+          <ol marker= "A." cols="1">
+            <li> sine </li>
+            <li> cosine </li>
+            <li> tangent </li></ol></p>
+      </statement>
+      <answer>
+        <p>
+          A
+        </p>
+      </answer>
+    </task>
+  </activity>
+
+  <activity xml:id="applying-trig-ratios">
+    <introduction>
+        <p>The top of the Eiffel Tower is seen from a distance of <m>d = 500</m> meters at an angle of <m>\alpha=31</m><degree/>.
+        </p>
+    </introduction> 
+    
+    <task>
+      <statement>
+        <p>
+          Draw a diagram to represent the situation. Use <m>x</m> to refer to the missing side.
+        </p>
+      </statement>
+      <answer>
+        <p>
+          Students should draw a right triangle where <m>\theta</m> is the angle formed with the ground, <m>500</m> meters as the side adjacent to <m>\theta</m>, and the height of the Eiffel Tower (side opposite <m>\theta</m>) as <m>x</m>.
+        </p>
+      </answer>
+    </task>
+    
+    <task>
+      <statement>
+        <p>
+          Which trig function could we use to find the height of the tower? 
+          <ol marker= "A." cols="1">
+            <li> sine </li>
+            <li> cosine </li>
+            <li> tangent </li></ol></p>
+      </statement>
+      <answer>
+        <p>
+          C
+        </p>
+      </answer>
+    </task>
+  
+    <task>
+      <statement>
+        <p>
+          How could we correctly set up the trigonometric ratio to find the height of the Eiffel Tower? 
+          <ol marker= "A." cols="2">
+            <li> <m>\sin{31}=\frac{x}{500}</m></li>
+            <li> <m>\cos{31}=\frac{500}{x}</m> </li>
+            <li> <m>\cos{31}=\frac{500}{x}</m> </li>
+            <li> <m>\tan{31}=\frac{x}{500}</m> </li></ol></p> 
```

Comment:
```suggestion
            <li> <m>\sin{31^\circ}=\frac{x}{500}</m></li>
            <li> <m>\cos{31^\circ}=\frac{500}{x}</m> </li>
            <li> <m>\cos{31^\circ}=\frac{500}{x}</m> </li>
            <li> <m>\tan{31^\circ}=\frac{x}{500}</m> </li></ol></p> 
```

Reply:
oh good! I was trying to use the <degree/> and that did not work! Thanks!

Reply:
Yea, the `<degree\>` is really only intended for text.  In math mode, just use this fairly standard LaTeX method.